mxnet-bot/MXNetCIBot/mxnet_ci_bot_lambda/.gitignore

```diff
@@ -0,0 +1,7 @@
+.serverless
+.idea
+.terraform
+terraform.tfstate
```

Comment:
Terraform?

Reply:
Lifted it from other .gitignores. Removing it since I don't use terraform for this.
Thanks for pointing out.